modules/test/playwright/pages/calendar-web/CalendarWidgetPage.ts

```diff
@@ -38,6 +38,7 @@ export class CalendarWidgetPage {
 	readonly timeZoneDropdown: Locator;
 	readonly title: Locator;
 	readonly toggleSideBarButton: Locator;
+	readonly untoggleSideBarButton: Locator;
```

Comment:
i think here it should be `toggledSideBarButton` and `untoggledSideBarButton`, because the way its written seems more like a action or a function, what do you think?

Reply:
Hi @rebsilva, good point! For me, I thought that having the 'Button' suffix made it clear that this is button linked to an action, as we have with other locators as well throughout our pages. But I see no problem in changing this! In this case, it would need to be the opposite, so, just to confirm:
		this.untoggledSideBarButton = page.locator(
			'.calendar-portlet-column-toggler .lexicon-icon-caret-left'
		);
		this.toggledSideBarButton = page.locator(
			'.calendar-portlet-column-toggler .lexicon-icon-caret-right'
		);

Reply:
as we agreed in person, it made more sense to stay like this in this specific case for the button we have in calendar that performs those actions. thanks Beca!! ❤️